app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -49,6 +50,13 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         revealInitialPrompt()
       })
 
+      this.$pageIsNotUsefulButton.on('click', function (e) {
+        if (window.GOVUK.cookie('_ga') === null) {
+          window.GOVUK.setCookie('_ga', '111111111.1111111111', {})
```

Comment:
I'm not sure we want to be setting the cookie to the dummy value - this may get confusing with our opt-in/opt-out method. I think explicitly setting a variable (as you were doing before) and passing this through to `setHiddenValuesNotUsefulForm()` is probably preferable.

Reply:
👍